.github/workflows/report-stable.yml

```diff
@@ -11,14 +11,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
What was the motivation here to add `fetch-depth` parameter?

Reply:
fetch-depth is needed to fetch the tags that the script uses to determine what version should be affected.

Reply:
I added comment to serve as a reminder

Reply:
Thanks.